src/components/pages/ArticlePage/Toegankelijkheid.tsx

```diff
@@ -0,0 +1,112 @@
+import type { FC } from 'react'
+
+import { Heading, Paragraph, List, ListItem, Link } from '@amsterdam/asc-ui'
+
+import BasePage from '../BasePage'
+
+const Toegankelijkheid: FC = () => (
+  <BasePage documentTitle="Toegankelijkheid" pageTitle="Toegankelijkheid">
+    <aside>
```

Comment:
The `aside` and `section` tags aren't really necessary imo. This is just an `article` with headings and `paragraph`s.